From 815cc98d6a7df142c8f1a9a2c1120650da278db0 Mon Sep 17 00:00:00 2001
From: Masamichi Hosoda <trueroad@trueroad.jp>
Date: Wed, 24 Aug 2016 21:27:32 +0900
Subject: Fix PostScript font alias name

`Helvetica Condensed' is not PostScript base 35 fonts.
`Helvetica Narrow' is PostScript base 35 fonts.

diff --git a/conf.d/30-metric-aliases.conf b/conf.d/30-metric-aliases.conf
index cd1e924..e72bf7a 100644
--- a/conf.d/30-metric-aliases.conf
+++ b/conf.d/30-metric-aliases.conf
@@ -9,7 +9,7 @@ Alias similar/metric-compatible families from various sources:
 PostScript fonts:       URW fonts:              GUST fonts:        Windows fonts:
 ======================  ======================  =================  ==================
 Helvetica               Nimbus Sans             TeX Gyre Heros
-Helvetica Condensed     Nimbus Sans Narrow      TeX Gyre Heros Cn
+Helvetica Narrow        Nimbus Sans Narrow      TeX Gyre Heros Cn
 Times                   Nimbus Roman            TeX Gyre Termes
 Courier                 Nimbus Mono             TeX Gyre Cursor
 ITC Avant Garde Gothic  URW Gothic              TeX Gyre Adventor
@@ -73,14 +73,14 @@ but in an order preferring similar designs first.  We do this in three steps:
 	<alias binding="same">
 	  <family>Nimbus Sans Narrow</family>
 	  <default>
-	  <family>Helvetica Condensed</family>
+	  <family>Helvetica Narrow</family>
 	  </default>
 	</alias>
 
 	<alias binding="same">
 	  <family>TeX Gyre Heros Cn</family>
 	  <default>
-	  <family>Helvetica Condensed</family>
+	  <family>Helvetica Narrow</family>
 	  </default>
 	</alias>
 
@@ -405,7 +405,7 @@ but in an order preferring similar designs first.  We do this in three steps:
 	</alias>
 
 	<alias>
-	  <family>Helvetica Condensed</family>
+	  <family>Helvetica Narrow</family>
 	  <default>
 	  <family>Arial Narrow</family>
 	  </default>
@@ -437,7 +437,7 @@ but in an order preferring similar designs first.  We do this in three steps:
 	<alias>
 	  <family>Arial Narrow</family>
 	  <default>
-	  <family>Helvetica Condensed</family>
+	  <family>Helvetica Narrow</family>
 	  </default>
 	</alias>
 
@@ -470,7 +470,7 @@ but in an order preferring similar designs first.  We do this in three steps:
 	</alias>
 
 	<alias binding="same">
-	  <family>Helvetica Condensed</family>
+	  <family>Helvetica Narrow</family>
 	  <accept>
 	  <family>TeX Gyre Heros Cn</family>
 	  <family>Nimbus Sans Narrow</family>
-- 
cgit v0.10.2

